your-code/main.py

```diff
@@ -1,6 +1,6 @@
 #1. Import the NUMPY package under the name np.
 
-
+asdf
```

Comment:
En este caso lo que tenías que hacer era un print(). Me preocupa mucho el nivel del ejercicio en este punto. 
```suggestion
print()
```